### ISSUE TITLE
Version "2" `docker-compose down` crashes when network doesn't exist

### ISSUE THREAD

**Issue by jzila** [docker/compose#3159]
Before version "2", `docker-compose down` was idempotent. Now it crashes if the default network doesn't exist.

My `docker-compose.yml` is the same as it was before, except for:
```
version: '2'
services:
    ...
```

```
$ docker-compose down
Removing network kbfsdocker_default
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/docker/client.py", line 142, in _raise_for_status
    response.raise_for_status()
  File "/usr/lib/python3.5/site-packages/requests/models.py", line 840, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localunixsocket/v1.22/networks/kbfsdocker_default

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/compose/cli/main.py", line 56, in main
    command.sys_dispatch()
  File "/usr/lib/python3.5/site-packages/compose/cli/docopt_command.py", line 23, in sys_dispatch
    self.dispatch(sys.argv[1:], None)
  File "/usr/lib/python3.5/site-packages/compose/cli/docopt_command.py", line 26, in dispatch
    self.perform_command(*self.parse(argv, global_options))
  File "/usr/lib/python3.5/site-packages/compose/cli/main.py", line 191, in perform_command
    handler(project, command_options)
  File "/usr/lib/python3.5/site-packages/compose/cli/main.py", line 273, in down
    project.down(image_type, options['--volumes'])
  File "/usr/lib/python3.5/site-packages/compose/project.py", line 229, in down
    self.networks.remove()
  File "/usr/lib/python3.5/site-packages/compose/network.py", line 152, in remove
    network.remove()
  File "/usr/lib/python3.5/site-packages/compose/network.py", line 79, in remove
    self.client.remove_network(self.full_name)
  File "/usr/lib/python3.5/site-packages/docker/utils/decorators.py", line 35, in wrapper
    return f(self, *args, **kwargs)
  File "/usr/lib/python3.5/site-packages/docker/api/network.py", line 40, in remove_network
    self._raise_for_status(res)
  File "/usr/lib/python3.5/site-packages/docker/client.py", line 145, in _raise_for_status
    raise errors.NotFound(e, response, explanation=explanation)
docker.errors.NotFound: 404 Client Error: Not Found ("b'network kbfsdocker_default not found'")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/docker-compose", line 9, in <module>
    load_entry_point('docker-compose==1.6.2', 'console_scripts', 'docker-compose')()
  File "/usr/lib/python3.5/site-packages/compose/cli/main.py", line 68, in main
    log_api_error(e)
  File "/usr/lib/python3.5/site-packages/compose/cli/main.py", line 89, in log_api_error
    if 'client is newer than server' in e.explanation:
TypeError: a bytes-like object is required, not 'str'
```

**Comment by dnephin** [comment]
Already fixed in #3135

**Comment by jzila** [comment]
Nice, thanks!
